client/src/cordova/build.action.mjs

```diff
@@ -155,13 +155,16 @@ async function androidDebug(verbose) {
     'WARNING: building "android" in [DEBUG] mode. Do not publish this build!!'
   );
 
+  // Alternatively:
+  // npx cordova run android -- --gradleArg=--include-build=../../src/cordova/android/OutlineAndroidLib
```

Comment:
This is a build script, doesn't seem to be relevant to running?

Reply:
I find it helpful to know the command we can use for troubleshooting and development.
I also think we should migrate to Taskfile and use the cordova command instead.